build.sbt

```diff
@@ -130,19 +128,49 @@ lazy val commonSettings = Seq(
 // General Settings
 inThisBuild(List(
   organization := "org.typelevel",
-  organizationName := "Typelevel Contributors",
-  developers := List(
-    Developer("rossabaker", "Ross A. Baker", "ross@rossabaker.com", url("https://github.com/rossabaker"))
-  ),
+  organizationName := "Typelevel",
+  publishGithubUser := "rossabaker",
+  publishFullName := "Ross A. Baker",
+  baseVersion := "0.3",
 
   homepage := Some(url("https://github.com/typelevel/case-insensitive")),
   startYear := Some(2020),
   licenses := Seq("Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.html")),
+  scmInfo := Some(ScmInfo(url("https://github.com/typelevel/case-insensitive"),
+    "git@github.com:typelevel/case-insensitive.git")),
 
-  pomIncludeRepository := { _ => false},
-  scalacOptions in (Compile, doc) ++= Seq(
-      "-groups",
-      "-sourcepath", (baseDirectory in LocalRootProject).value.getAbsolutePath,
-      "-doc-source-url", "https://github.com/typelevel/case-insensitive/blob/v" + version.value + "€{FILE_PATH}.scala"
+  githubWorkflowTargetTags ++= Seq("v*"),
+  githubWorkflowPublishTargetBranches := Seq(RefPredicate.StartsWith(Ref.Tag("v"))),
+  githubWorkflowBuildPreamble := Seq(
+    WorkflowStep.Use("olafurpg", "setup-gpg", "v3"),
+    WorkflowStep.Use("actions", "setup-ruby", "v1"),
+    WorkflowStep.Run(List("""
+      |gem install bundler
+      |bundle install --gemfile=site/Gemfile
+    """.stripMargin), name = Some("Install Jekyll"))
```

Comment:
This is super slow.  Maybe we can manage this from nix?

Reply:
Also worth noting that you can just do `List("gem install bundler", "bundle install --gemfile=site/Gemfile")` rather than manually fiddling with newlines.

But aside from that, this seems slower than just installing the gems directly. I would make this conditional on the `matrix.scala` versions that you're building the microsite for, at the very least.

Reply:
This trick was necessary for repeatable builds on cats-effect.  I don't know an incantation that's repeatable without either bundler or nix.